### PR TITLE
feat: CLI can generate tech radar rendering from specification files

### PR DESCRIPTION
**Pull request by Alex-itu**
This feature integrates Thoughtworks tech radar into our project. The code has been refactored to better fit in the go cli tool. 

Note:
The code is very big and will be worked on in the future.
For reviewers, focus on the following files:  
- MakeHtml lines: 52-57
- Skim factory.js
and skip style.css

Resolves. [#34](https://github.com/orgs/NovoNordisk-OpenSource/projects/2/views/3?pane=issue&itemId=53689360)

### CODE REVIEW

File: docs/dev_docs/read_and_write.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Technologies for the radar (or blips) are stored in the program using structs. T
 For reading spec files in csv format, the `ReadCsvSpec` function is used. Since we know the structure of our data we can predefine our structs to fit that structure.  
 By using the [gocarina/gocsv](https://github.com/gocarina/gocsv) module we can  unmarshal the csv file. Taking the csv file and turning it into our data structure.
 
+For reading spec files in csv format and then change it into a string, the `CsvToString` is used. The purpose is for giving the csv data to the javascripts. The data given is then injected into html. This data is used for the javascript and is therefor hidden in the html.
```
Comment (Haruyzal) [inline docs/dev_docs/read_and_write.md:11]: I'm confused by what is meant by:

> The purpose is for giving the csv data to the **javascripts**.

File: src/js/util/factory.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  'd3',
+  'lodash',
+  '../models/radar.js',
+  '../models/quadrant.js',
+  '../models/ring.js',
+  '../models/blip.js',
+  '../graphing/radar.js',
+  '../config.js',
+  '../graphing/config.js'
+], function(d3, _, Radar, Quadrant, Ring, Blip, GraphingRadar, configFuntion, config) {
```
Comment (Haruyzal) [inline src/js/util/factory.js:11]: Just outta curiosity, what's d3 and _? :)

**Review by Haruyzal — APPROVED** [review]
Looks really great! Well fkn done o.o

Think I've got some questions for clarification tho :3

File: src/js/util/factory.js (hunk 2 above)
Comment (JonasSkjodt) [reply src/js/util/factory.js:11]: they are both js libraries, d3 is a statistics javascript library which is a "open-source JavaScript library for visualizing data" and lodash (_) "makes JavaScript easier by taking the hassle out of working with arrays, numbers, objects, strings, etc." We refer to them by name after defining them in a config file.

**Review by August-Brandt — APPROVED** [review]
Look greate

**Pull request merged**